packages/rpc/src/index.ts

```diff
@@ -25,6 +26,12 @@ export * from './methods/bitcoin/send-transfer';
 export * from './methods/bitcoin/sign-message';
 export * from './methods/stacks/stx-sign-message';
 export * from './methods/stacks/stx-sign-transaction';
+export * from './methods/stacks/stx-contract-call';
```

Comment:
Can you rename this file here? It should match the method name being called? `stx-call-contract.ts`